connect/eaas/runner/helpers.py

```diff
@@ -86,3 +92,40 @@ def to_ordinal(val):
     if val > 14:
         return f"{val}{ORDINAL_SUFFIX.get(int(str(val)[-1]), 'th')}"
     return f"{val}{ORDINAL_SUFFIX.get(val, 'th')}"
+
+
+def get_connect_version():
```

Comment:
Please use ConnectClient for consistency.
If you get a ClientError you can check the returned http status.
The output message `API Key is not valid` should be printed only in case of 401/403.
If no exception is raised you can get the header the following way:

`client.response.headers['Connect-Version']`